<launch>
  <node pkg = "iqr_ks106_uart" type = "ks106_uart_node" name = "ks106_uart" output = "screen">
    <param name = "ks106_pub_topic" type = "string" value = "Ks106_uart"/>
    <param name = "uart_port" type = "string" value = "/dev/ks106_link"/>
    <param name = "baudrate" type = "int" value = "9600"/>
    <param name = "frequency" type = "int" value = "40"/>
    <param name = "frame_id_1" type = "string" value = "test_1"/>
    <param name = "frame_id_2" type = "string" value = "test_2"/>
    <param name = "frame_id_3" type = "string" value = "test_3"/>
    <param name = "frame_id_4" type = "string" value = "test_4"/>
  </node>
</launch>